.changeset/silly-forks-nail.md

```diff
@@ -0,0 +1,12 @@
+---
+"@salt-ds/core": minor
+---
+
+Added "code" option to Text 'styleAs' prop
+
+```tsx
+<>
+  <Code>Code text</Code>
+  <Text styleAs="code">Text styled as code</Text>
+</>
+```
```

Comment:
Realised we've had a changelog like this before so we should just copy it :)

```suggestion
`Text` has a new variant: `Code`. See [Text component](/salt/components/text) for more information.

```tsx
<>
  <Code>Code text</Code>
  <Text as="code">Text styled as code</Text>
  <Text styleAs="code">Text styled as code</Text>
</>
```
```